src/components/Hero/Hero.stories.tsx

```diff
@@ -353,19 +372,154 @@ export const CampaignBackgroundColors: Story = {
         <Hero
           backdropBackgroundColor="section.education.primary"
           backgroundColor="ui.warning.primary"
-          foregroundColor="ui.typgraphy.heading"
+          foregroundColor="ui.typography.heading"
+          heroType="campaign"
+          heading={
+            <Heading
+              level="h1"
+              id="campaign-hero-long-text-heading"
+              text="Hero Campaign"
+            />
+          }
+          imageProps={imageProps}
+          subHeaderText={otherSubHeaderTextLong}
+        />
+      </div>
+    </Stack>
+  ),
+};
+export const TextColorStyles: Story = {
+  render: () => (
+    <Stack spacing="l">
+      <div>
+        <Heading
+          id="campaign-hero-default"
+          size="heading6"
+          text="Campaign Hero with Default Colors"
+        />
+        <Hero
+          backgroundImageSrc="//loremflickr.com/1600/800/kitten"
+          heroType="campaign"
+          heading={
+            <Heading
+              level="h1"
+              id="campaign-hero-default-heading"
+              text="Hero Campaign"
+            />
+          }
+          imageProps={imageProps}
+          isDarkBackgroundImage
+          subHeaderText={otherSubHeaderText}
+        />
+        <Source
```

Comment:
We've been putting the `Source` component in mdx files but this seems to make more sense -- to be colocated with the code.

Reply:
The "Show code" button still shows up and can make it confusing. I think it can be disabled, can you look into that?